### PR TITLE
[Chat] Remove inline image from stable

### PR DESCRIPTION
**Pull request by Leah-Xia-Microsoft**
# What

Since we are not release Inline Image GA in 1.12.0, removing this feature from main

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/chat-component-bindings/src/handlers/createHandlers.ts [hunk 1]
```diff
@@ -68,7 +68,7 @@ export const createDefaultChatHandlers = memoizeOne(
         }
       ) => {
         const updatedMetadata = options?.metadata ? { ...options.metadata } : {};
-        updatedMetadata.fileSharingMetadata = JSON.stringify(options?.attachmentMetadata || []);
+        updatedMetadata['fileSharingMetadata'] = JSON.stringify(options?.attachmentMetadata || []);
```
Comment (vhuseinova-msft) [inline packages/chat-component-bindings/src/handlers/createHandlers.ts:71]: We probably don't need to revert this

File: packages/chat-component-bindings/src/messageThreadSelector.ts [hunk 2]
```diff
@@ -61,7 +60,7 @@ const memoizedAllConvertChatMessage = memoizeFnAll(
 
 /* @conditional-compile-remove(file-sharing) */
 const extractAttachedFilesMetadata = (metadata: Record<string, string>): FileMetadata[] => {
-  const fileMetadata = metadata.fileSharingMetadata;
+  const fileMetadata = metadata['fileSharingMetadata'];
```
Comment (vhuseinova-msft) [inline packages/chat-component-bindings/src/messageThreadSelector.ts:63]: We probably don't need to revert this

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx [hunk 3]
```diff
@@ -7,14 +7,14 @@ import { mergeClasses } from '@fluentui/react-components';
...
+
 import { editBoxStyle, inputBoxIcon, editingButtonStyle, editBoxStyleSet } from '../styles/EditBox.styles';
 import { InputBoxButton, InputBoxComponent } from '../InputBoxComponent';
 import { MessageThreadStrings } from '../MessageThread';
 import { useChatMyMessageStyles } from '../styles/MessageThread.styles';
 import { ChatMessage } from '../../types';
 import { _FileUploadCards } from '../FileUploadCards';
-/* @conditional-compile-remove(file-sharing) */
-import { FileMetadata } from '../FileDownloadCards';
+import { AttachmentMetadata, FileMetadata } from '../FileDownloadCards';
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/ChatMessage/ChatMessageComponentAsEditBox.tsx:17]: Do we need to have CC here?

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx [hunk 4]
```diff
@@ -156,8 +157,8 @@ const MessageBubble = (props: ChatMessageComponentAsMessageBubbleProps): JSX.Ele
     onRenderAvatar,
     showMessageStatus,
     messageStatus,
-    /* @conditional-compile-remove(file-sharing) */
     fileDownloadHandler,
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx:160]: just wondering if @conditional-compile-remove(teams-inline-images-and-file-sharing) missed here?

File: packages/react-components/src/components/MessageThread.test.tsx [hunk 5]
```diff
@@ -8,13 +8,16 @@ import { MessageThread } from './MessageThread';
...
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
 import { AttachmentDownloadResult, AttachmentMetadata } from './FileDownloadCards';
 import { createTestLocale, renderWithLocalization } from './utils/testUtils';
 /* @conditional-compile-remove(date-time-customization) @conditional-compile-remove(data-loss-prevention) */
 import { COMPONENT_LOCALE_EN_US } from '../localization/locales';
 /* @conditional-compile-remove(date-time-customization) */
 import { screen } from '@testing-library/react';
+/* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
 import { render, waitFor } from '@testing-library/react';
+/* @conditional-compile-remove(data-loss-prevention) */ /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/MessageThread.test.tsx:20]: Do we need to have /* @conditional-compile-remove(data-loss-prevention) */ here?

File: packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx [hunk 6]
```diff
@@ -18,8 +20,11 @@ import {
   TypingIndicatorStylesProps,
   useTheme
 } from '@internal/react-components';
+/* @conditional-compile-remove(image-gallery) */
 import { ChatMessage } from '@internal/react-components';
```
Comment (vhuseinova-msft) [inline packages/react-composites/src/composites/ChatComposite/ChatScreen.tsx:24]: It looks like it's used also in file sharing

File: packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts [hunk 7]
```diff
@@ -34,11 +34,14 @@ import {
 } from './ChatAdapter';
 import { AdapterError } from '../../common/adapters';
 /* @conditional-compile-remove(file-sharing) */
-import { FileUploadAdapter, convertFileUploadsUiStateToMessageMetadata } from './AzureCommunicationFileUploadAdapter';
-/* @conditional-compile-remove(file-sharing) */
-import { AzureCommunicationFileUploadAdapter } from './AzureCommunicationFileUploadAdapter';
+import {
+  FileUploadAdapter,
+  AzureCommunicationFileUploadAdapter,
```
Comment (vhuseinova-msft) [inline packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts:39]: Is /* @conditional-compile-remove(file-sharing) */ or /* @conditional-compile-remove(teams-inline-images-and-file-sharing) */ missed here?

File: packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx (hunk 4 above)
Comment (JoshuaLai) [reply packages/react-components/src/components/ChatMessage/ChatMessageComponentAsMessageBubble.tsx:165]: onInlineImageClicked is just used in the case of image gallery so this should be ok

**Review by JoshuaLai — APPROVED** [review]

File: packages/react-components/src/components/MessageThread.test.tsx (hunk 5 above)
Comment (Leah-Xia-Microsoft) [reply packages/react-components/src/components/MessageThread.test.tsx:20]: We need the data-loss prevention one because it's used in line 145

**Pull request merged**